static/docs/get-started/import-data.md

```diff
@@ -0,0 +1,70 @@
+# Import Data
+
+We've seen how to [push](/doc/get-started/store-data) and
+[pull](/doc/get-started/retrieve-data) data a <abbr>DVC project</abbr>'s
+[remote](/doc/commands-reference/remote). But what if we wanted to integrate a
+dataset or ML model produced in one project into another project? (As an
+[external dependency](/doc/user-guide/external-dependencies))
+
+One way is to download the data (with `wget` or `dvc get`, for example) and use
+`dvc add` to track it locally, but the connection between projects would be
+lost: we couldn't tell where the data came from or whether there are new
+versions available. We offer the `dvc import` command instead (for projects
+hosted on a Git repository). It downloads the data, and creates a
+[DVC-file](/doc/user-guide/dvc-file-format) to track changes in the imported
+data.
+
+<details>
```

Comment:
I think it went to far a bit :) Let's show the command at least with some, may be with this intro: `In the Add Files chapter we download raw data using the dvc get command, and then track it with dvc add. Let's see how this would be improved by, instead, just running:` you have

Reply:
Show the command with some ?
I have ?

Reply:
I think I may know what you mean. It's too detailed. I slashed it by half now. Better?

Reply:
Pushed new version we worked on together.